app/src/ui/changes/changes-list.tsx

```diff
@@ -463,6 +483,41 @@ export class ChangesList extends React.Component<
     )
   }
 
+  private onStashEntryClicked = () => {
+    if (this.props.isShowingStashEntry) {
+      this.props.dispatcher.hideStashEntry(this.props.repository)
+    } else {
+      this.props.dispatcher.showStashEntry(this.props.repository)
+    }
+  }
+
+  private renderStashedChanges() {
+    if (this.props.stashEntry === null) {
```

Comment:
Feature flag

Reply:
Seems to have been resolved on line 495.